app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Ushahidi\App\Exceptions;
+
+use Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException;
+
+/**
+ * Validation Exception
+ */
+class ValidationException extends UnprocessableEntityHttpException
```

Comment:
I needed a way to catch the errors from the Validators and return them in the HTTP response - we did something similar in Kohana. Hopefully we can come up with a cleaner way later on, this was a bit ugly.